crates/css_ast/src/values/flexbox/mod.rs

```diff
@@ -114,4 +114,4 @@ pub enum AlignSelfStyleValue {}
 #[percentages("n/a")]
 #[canonical_order("per grammar")]
 #[animation_type("discrete")]
-pub enum AlignContentStyleValue {}
+pub enum FlexAlignContentStyleValue {}
```

Comment:
@keithamus I wasnt too sure here. The names had a conflict with what css_ast::values::align was trying to expose. Not sure which is the canonical truth, or how we handle conflicts like this. Just prepended to keep moving, but fully open to your advice.